### PR TITLE
10328: Update tests for per patient bulk ordering

### PR DESCRIPTION
**Pull request by PhysicsJon**
[AB#10328](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10328)
[AB#9985](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9985)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Actions/Pages/OrganisationsOrdersDashboard.cs [hunk 1]
```diff
@@ -16,14 +16,14 @@ public OrganisationsOrdersDashboard(IWebDriver driver) : base(driver)
...
-            Wait.Until(d => CreateNewOrderButtonDisplayed());
+            //Wait.Until(d => d.FindElements(Pages.OrganisationsOrdersDashboard.PageTitle).Count == 1);
+            //Wait.Until(d => CreateNewOrderButtonDisplayed());
         }
 
         public void SelectExistingOrder(string callOffAgreementId)
         {
             Wait.Until(d => d.FindElements(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)).Count == 1);
-            Wait.Until(ElementExtensions.ElementToBeClickable(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)));
+            //Wait.Until(ElementExtensions.ElementToBeClickable(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)));
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Actions/Pages/OrganisationsOrdersDashboard.cs:26]: What's your plan with the commented-out lines?

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature [hunk 2]
```diff
@@ -318,7 +318,7 @@ Scenario Outline: Associated Service - edit price screen - Flat price All data a
 	And fills in the Associated Service edit form with valid data
 	When the User chooses to save
 	Then the Associated Service is saved
-	And the Associated Services dashboard is presented
+	#And the Associated Services dashboard is presented
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature:321]: What's your plan with the commented out line?

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature [hunk 3]
```diff
@@ -446,10 +432,10 @@ Scenario Outline: Catalogue Solutions - edit price screen - Flat price Go back a
     And the Catalogue Solution is saved in the DB
 
     Examples:
-        | ProvisioningType |
-        #| declarative      |
-        #| variable         |
-        | per patient      |
+        | ProvisioningType | quantityLabel |
+        # | declarative   |  |
+        # | variable      |  |
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature:437]: I guess you're waiting for the relevant FE work to be completed and will gradually reinstate the commented out provisioning types?

File: src/OrderFormAcceptanceTests.Actions/Pages/OrganisationsOrdersDashboard.cs (hunk 1 above)
Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrganisationsOrdersDashboard.cs]: I've pushed a commit that should have uncommented them

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature (hunk 3 above)
Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature:437]: Indeed that is the plan

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature [hunk 4]
```diff
@@ -252,7 +252,7 @@ Scenario Outline: Associated Service - edit price screen - Flat price Data type
 	Given there are no Service Recipients in the order
 	And the User is presented with the Associated Service edit form for a <ProvisioningType> flat price
 	And fills in the Associated Service edit form with valid data
-	And the quanitity is a decimal
+	And the quantity quanitity is a decimal
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature:255]: Should `quanitity` be `quantity`?

I think the doubling up of `quantity` might look like a mistake at first to most people. Is there a way of tweaking the wording of the step slightly to make it a bit clearer?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature]: Agreed, see latest commit

**Review by bjss-chris — APPROVED** [review]
👍

**Pull request merged**